lib/cassandra/cluster/schema/cql_type_parser.rb

```diff
@@ -72,6 +72,9 @@ def lookup_type(node, types)
             Cassandra::Types.tuple(*node.children.map { |t| lookup_type(t, types)})
           when 'empty'             then
             Cassandra::Types.custom('org.apache.cassandra.db.marshal.EmptyType')
+          when /^'/ then
```

Comment:
use `\A` instead of `^` here, `\A` signifies start of string, while `^` is start of line, not likely to affect this particular use-case, but more correct in general (http://stackoverflow.com/questions/3632024/why-do-rubys-regular-expressions-use-a-and-z-instead-of-and)